helpers/html/render_html.go

```diff
@@ -13,17 +19,22 @@ func RenderWithError(c *gin.Context, status int, view string, title string, err
 	})
 }
 
-func RenderWithNotice(c *gin.Context, status int, view string, title string, notices []interface{}, data map[string]interface{}) {
+func RenderWithFlash(c *gin.Context, status int, view string, title string, data map[string]interface{}) {
+	flashKey, flashValue := getFlashMessage(c)
+
 	ginview.HTML(c, status, view, gin.H{
-		"title":   title,
-		"notices": notices,
-		"data":    data,
+		"title":  title,
+		flashKey: flashValue,
```

Comment:
This implementation is quite complex:
- It relies on `getFlashMessage`
- It requires to import `html "github.com/gutakk/go-google-scraper/helpers/html"` each time a new flash message must be added to use the constants `FlashNoticeKey` and `FlashErrorKey`

How about instead:

- Add a flash key when adding a new Flash:

Session.addFlash accepts a second parameter: https://github.com/gorilla/sessions/blob/master/sessions.go#L65

So it's possible to do:

```
session.AddFlash("This is an info message", "notice")
session.AddFlash("This is an error message", "error")
```
 
- Keeping the HTML variable to `flashes` (or `errors`?) and `notices` 

```
ginview.HTML(c, status, view, gin.H{
  "title": title,
  "errors": session.Flashes("error"),
  "notices": session.Flashes("notice")
  "data": data,
})
```

Then in the HTML template, iterate over the HTML variables for the content of each:

```
{{range .errors}}{{.}}{{end}}
{{range .notices}}{{.}}{{end}}
```

Reply:
Fixed in 01e1b958ff94cdaf5702b9fb683bdc5c84cce769

Reply:
Looks neat 💪

Reply:
Now after going through the code 🙈 an improvement I still can see is to have:

- `session.AddNoticeFlash(c, logoutSuccessFlash)` instead of `session.AddFlash(c, logoutSuccessFlash, "notice")`
- `session.AddErrorFlash(c, loginErrir)` instead of `session.AddFlash(c, logoutSuccessFlash, "error")`

This can be done in the next PR too 😁